merlin-server/sql/merlin/tables/plan_snapshot.sql

```diff
@@ -8,12 +8,26 @@ create table plan_snapshot(
   plan_id integer
     references plan
     on delete set null,
-
   revision integer not null,
-  name text not null,
```

Comment:
Are these breaking changes?

Reply:
No, because nobody besides the DB was able to interact with this table, and the changes have been propagated across the DB

Reply:
It looks like these were publicly exposed over GQL for regular users in https://github.com/NASA-AMMOS/aerie/pull/1029, so a name change might still count as breaking, not sure 🤔

Reply:
It's not a name change, though. The `name` column (which stored the plan name) is getting replaced with `snapshot_name.

Reply:
Either way, I would imagine removing columns that users might have been querying would still count as a breaking change.

Reply:
Regardless, I know for certain that nobody is currently using this table outside the DB, regardless of its exposure in #1029 ([if you trace it's history you'll see it was accidentally tracked way back when Plan Merge was initially implemented](https://github.com/NASA-AMMOS/aerie/pull/399/files#diff-179570744d2b8431068cbac1cc2a645e1ce9d45226b88d720af05b471bf2896f)). If you feel strongly on this change being breaking, however, feel free to add the `breaking change` label.

Reply:
Philosophically it might not feel like a breaking change, but technically I would argue that it meets the definition and should be recorded as accordingly in our e.g. upgrade docs, just in case.

Reply:
We can check with @camargo when we do the release next week if this should be included in the upgrade docs.